package.json

```diff
@@ -42,7 +42,7 @@
     "postcss-value-parser": "^3.3.0",
     "stdin": "0.0.1",
     "stylelint": "^7.5.0",
-    "stylelint-order": "^0.3.0"
+    "stylelint-order": "0.4.x"
```

Comment:
I'm not expecting any breaking changes anymore in stylelint-order. But for precaution matter I support @doochik's fixating version number.